### ISSUE TITLE
Time Offset in the OSD File...

### ISSUE THREAD

**Issue by AxelRed65** [avsaase/walksnail-osd-tool#51]
### Is there an existing feature request for this?

- [X] I have searched the existing open and closed issues

### Feature description

Hi, I have noticed that it seems the original Video from the VTX video starts with an offset of few seconds from the video ( and OSD ) recorded on the googles, I believe googles start recordin at arming moment, while the VTX @ takeoff moment... So, using the OSDTool to mix the OSD file with the original VTX Video is almost impossible without modifying the OSD file itself...
I think it would be nice to have an offset feature to tune up where the OSD file starts to be 'mixed' with the video, to make the alignment between VTX Video and OSD file possible....

Great work in any case until now...

regards

Alex

**Comment by JebKerman91** [comment]
I agree that an adjustment to better line up the OSD with the VTX video is needed - I'm not sure if the mis-alignment happens as described, at least in my case where I manually command recording.

Still, e.g. a slider that would make it possible to adjust the OSD timing +-3 seconds with 0.1 seconds of resolution would work.

**Comment by GregoryMaks** [comment]
+- 3 seconds may not be enough. I have OSD lagging at least 50-60s on my new Goggles L :(

**Comment by rickgode** [comment]
Yeah, this would be FANTASTIC.  Especially on really long videos (25-30min)  It's easy to tell because my AHI is off (maybe about 20 secs?)

Thanks!

**Comment by avsaase** [comment]
I don't think it's enough to just add a fixed offset. There's no guarantee that the goggle OSD and vtx recordings stay in sync because frames may be dropped in the downlink. If you want to correct for that as well you need to resync the OSD at several points throughout the video and I think functionality like that is better left to a video editor. I suggest rendering the OSD with a chroma key.
